doc/design.txt

```diff
@@ -210,15 +210,15 @@ An image trailer has the following structure:
      0                   1                   2                   3
```

Comment:
There is nothing wrong with this line, but the GitHub UI won't let me comment on parts of the file that aren't in a diff hunk.

It looks like the "BOOT STATES" section above also needs to be reworked in light of the recent changes as well.